.vault-config/productconstructionint.yaml

```diff
@@ -0,0 +1,14 @@
+storageLocation:
+  type: azure-key-vault
+  parameters:
+    subscription: cab65fc3-d077-467d-931f-3932eabf36d3
+    name: ProductConstructionInt
+
+secrets:
+  build-asset-registry-sql-connection-string:
+    type: sql-connection-string
+    parameters:
+      adminConnection: build-asset-registry-admin-connection-string
+      dataSource: tcp:maestro-int-server.database.windows.net,1433
+      database: BuildAssetRegistry
+      permissions: rw
```

Comment:
I am not sure we'll ever need the `w`. Maybe it's safer we only have `r` for now

Reply:
makes sense, removed the `w`

Reply:
Can you regenerate the secret too?

Reply:
Actually, I think it might be better to not have this secret at all, using the `DefaultAzureCredential`. Then we'd just have the passwordless connection string in the config

Reply:
I'll give it a try quickly

Reply:
Apparently this is supported in .NET 7+ only, so we can't use it yet. I'll create a follow up issue

Reply:
I rotated the secret with only the reader rights